raceSystem/webUI.py

```diff
@@ -0,0 +1,194 @@
+from flask import Flask, render_template, request, jsonify
+from raceSystem.userInterface import UserInterface
+
```

Comment:
Flask print a lot to the console, this is not needed in our main branch and it somehow deletes x number of older lines in the command line output making the last raceresult disappear. Can you add the following lines of code:
import logging
log = logging.getLogger('werkzeug')
log.setLevel(logging.ERROR)
This will make flask only log errors in the console

Reply:
Done, Flask now only print error messeges in the console.